src/components/modules/blockSelection.ts

```diff
@@ -17,6 +17,14 @@ import { SanitizerConfig } from '../../../types/configs';
  *
  */
 export default class BlockSelection extends Module {
+
+  /**
+   * Saves any block selected cache
```

Comment:
```suggestion
   * Sometimes .anyBlockSelected can be called frequently, 
   * for example at ui@selectionChange (to clear native browser selection in CBS)
   * We use cache to prevent multiple iterations through all the blocks
```